DEVELOPING.md

```diff
@@ -62,7 +62,8 @@ rule is that minors are breaking changes and patches are new features).
 1. `git commit -a -m 'Update docs for release'`
 1. `npm config set tag-version-prefix ''`
 1. `npm version minor -m 'rel: %s'` (replace `minor` with `patch` if no breaking changes)
-1. Build npm packages and publish them: `bazel run //internal:npm_package.publish && bazel run //internal/karma:npm_package.publish`
+1. Create a clean directory to use as output: `rm -rf ./rules_typescript_release && mkdir ./rules_typescript_release`
+1. Build npm packages and publish them: `bazel run //internal:npm_package.publish --config=release --output_base=rules_typescript_release && bazel run //internal/karma:npm_package.publish --config=release --output_base=rules_typescript_release`
 1. `bazel build :release`
```

Comment:
@alexeagle @mgechev this step wasn't here when I looked at the instructions before, and I am not sure if it affects/is affected by the steps I'm adding. Can you have a look please?